starter-core/src/main/java/io/micronaut/starter/feature/github/workflows/Secret.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.github.workflows;
+
+/**
+ * GitHub secret.
+ *
+ * @author Pavol Gressa
+ * @since 2.2
+ */
+public class Secret {
```

Comment:
The purpose of having this object here is to automatically create the secrets using starter api